### PR TITLE
Api

### PR DESCRIPTION
**Pull request by joelethan**
**_What does this PR do?_**
1. Create a model for the api. These are the classes holding the logic operation of the api
2. Create endpoints for the api. The endpoints created in this PR are 
   - Place an order for food
   - Get a list of all orders
   - Get a specific order
   - Update the status of an order
   - Delete a specific order
3. Create tests for the endpoints and model of the api
4. Add badges to show the test-coverage 
[Build Status](https://travis-ci.org/joelethan/Fast-Food-Fast) [Coverage Status](https://coveralls.io/github/joelethan/Fast-Food-Fast?branch=api)

**_What are the relevant pivotal tracker stories?_**
#160692468
#160689850
#160689975
#160690457
#160692058

**_What is the link to the api branch?_**
https://github.com/joelethan/Fast-Food-Fast/tree/api

**_What is the link to the api hosted on heroku?_**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,37 @@
+astroid==2.0.4
```
Comment (jeanjoe) [inline requirements.txt]: Nice work flow, Not sure if u used all the modules in the requirements.txt file, you can remove the ones your app doesn't use.

File: api/model.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+            return False
+
+    def delete_order(self, order_id):
+        for order in self.orders_list:
+            if order['order_id']==order_id:
+                self.orders_list.remove(order)
+                return order
+            return False
+ 
+    def find_specific_order(self, foodname):
```
Comment (kdaglas) [inline api/model.py]: btw this is also good... a customer might want to use the food name to get that order... though you need some really good validation for that... but i like the idea

**Comment by kdaglas** [comment]
short and precise... good... though seems you didnt fully test everything... your coverage is good but can be better

**Comment by kdaglas** [comment]
i like the pull request description... its on point

**Pull request merged**